### ISSUE TITLE
My SDXL LoRA training loss always be nan, how can I fix it?

### ISSUE THREAD

**Issue by neverbiasu** [Akegarasu/lora-scripts#508]
Here's my latest training config.
I've tried using fixed version vae, AdaFactor optimizer, no_half_vae=True, mixed_precision = "bf16"... 
Still cannot find a solution, please help me if u have an idea!
```toml
pretrained_model_name_or_path = "D:/sd-webui-aki/sd-webui-aki-v4.5/models/Stable-diffusion/ponyDiffusionV6XL_v6StartWithThisOne.safetensors"
vae = "D:/sd-webui-aki/sd-webui-aki-v4.5/models/VAE/sdxl.vae.safetensors"
train_data_dir = "./train/yanjun_sdxl"
prior_loss_weight = 1
resolution = "1024,1024"
enable_bucket = false
min_bucket_reso = 256
max_bucket_reso = 1024
bucket_reso_steps = 64
output_name = "yanjun_XL"
output_dir = "./output"
save_model_as = "safetensors"
save_precision = "fp16"
save_every_n_epochs = 2
max_train_epochs = 10
train_batch_size = 1
gradient_checkpointing = false
network_train_unet_only = true
network_train_text_encoder_only = false
learning_rate = 1e-5
unet_lr = 1e-5
text_encoder_lr = 1e-6
lr_scheduler = "cosine_with_restarts"
lr_warmup_steps = 0
lr_scheduler_num_cycles = 1
optimizer_type = "AdaFactor"
network_module = "networks.lora"
network_dim = 128
network_alpha = 64
log_with = "wandb"
logging_dir = "./logs"
caption_extension = ".txt"
shuffle_caption = false
weighted_captions = false
keep_tokens = 0
max_token_length = 255
caption_tag_dropout_rate = 0
seed = 1337
mixed_precision = "bf16"
full_fp16 = false
full_bf16 = false
fp8_base = false
no_half_vae = true
xformers = true
lowram = false
cache_latents = true
cache_latents_to_disk = true
cache_text_encoder_outputs = true
persistent_data_loader_workers = true

```

**Comment by wzgrx** [comment]
+1

**Comment by Akegarasu** [comment]
update to latest version
